e2e-tests/cypress/support/core-page-checks.js

```diff
@@ -37,8 +37,6 @@ const checkBackLink = (currentHref, expectedHref) => {
   cy.url().should('eq', expectedUrl);
 
   // go back to the current page
-  // cy.navigateToUrl(`${Cypress.config('baseUrl')}${currentHref}`);
```

Comment:
is there a reason this is commented out and not removed?

Reply:
This is strange - it's not commented out, but github PR is showing that it is. I've even pushed some commits changing this file and it's still showing this commented line. Viewing the file by itself inside this PR shows the latest 🤷‍♂️  https://github.com/UK-Export-Finance/exip/blob/540b66f8f1aab43743025cc760f6c50bdb09a6b9/e2e-tests/cypress/support/core-page-checks.js